Lesson13/Main.java

```diff
@@ -0,0 +1,97 @@
+package Lesson13;
+
+import java.util.*;
+
+public class Main {
+    public static void main(String[] args) {
+
+        /* Task 1: Обрати текст, записати його в змінну типу String. Кожне слово цього тексту помістити в колекцію.
+        Вивести в консоль: загальну кількість слів. Вивести елементи колекції у виді (З допомогою ітератора).
+        Кожен елемент колекції довжина якого більше 6, видалити цей елемент, а для елементів які менше 3 символів
+        додати в кінець таких символів ’*’ щоб довжина була 3. Вивести колекцію (з допомогою for-each).
+         */
+        String str = "Дано N цілих чисел. Знайти суму трицифрових чисел, що є серед даного набору чисел";
+        List<String> list = new ArrayList<>(List.of(str.split(" ")));
```

Comment:
я б отут додав ще отакий рядок щоб із твоєї стрінги видалити всі завйві символи 
```suggestion
         */
        String str = "Дано N цілих чисел. Знайти суму трицифрових чисел, що є серед даного набору чисел";
        str = str.replaceAll("[^a-zA-Z ]", "");
        List<String> list = new ArrayList<>(List.of(str.split(" ")));
```

Reply:
Додано,
дякую :)